### PR TITLE
Create api.py

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: pythonAPI/api.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+"""
```
Comment (shikhagupta1212) [inline pythonAPI/api.py]: rename file. Not intuitive what is api?

File: pythonAPI/enable_logging_api.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    Parameters:
+        project_number (int):Project Number of the cloud project
+        token (str):The authorization token for cloud Project
+
+    Returns:
+        boolean: whether logging API is enabled or not
+    """
+    name = "projects/" + project_number + "/services/logging.googleapis.com"
+    url = "https://serviceusage.googleapis.com/v1/" + name + ":enable"
```
Comment (shikhagupta1212) [inline pythonAPI/enable_logging_api.py:18]: Can you check what will happen if you try to enable the API which is already enabled?

File: pythonAPI/api.py (hunk 1 above)
Comment (siddhimahawar) [reply pythonAPI/api.py]: This file is the one in which api is created
what should I name it?

File: pythonAPI/enable_logging_api.py (hunk 2 above)
Comment (siddhimahawar) [reply pythonAPI/enable_logging_api.py:18]: If it's enabled already then no change in state is there

**Review by piy314 — APPROVED** [review]

File: pythonAPI/api.py (hunk 1 above)
Comment (piy314) [reply pythonAPI/api.py]: I have approved, but rename this file. Say "admin_app.py" or "admin_service.py"

**Pull request merged**